sdk/typescript/src/utils/constants.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { normalizeSuiObjectId } from './sui-types.js';
+
+export const SUI_DECIMALS = 9;
+export const MIST_PER_SUI = BigInt(1000000000);
+
+export const MOVE_STDLIB_ADDRESS = '0x1';
+export const SUI_FRAMEWORK_ADDRESS = '0x2';
+export const SUI_SYSTEM_ADDRESS = '0x3';
+export const SUI_CLOCK_OBJECT_ID = normalizeSuiObjectId('0x6');
```

Comment:
I know this is copied over but it's weird that this is normalized and the others are not